projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -302,9 +444,16 @@
   "quark.misc.my_son": "不要再跟我和我的儿子说话了。",
   "quark.misc.my_daughter": "不要再跟我和我的女儿说话了。",
   "quark.misc.my_child": "不要再跟我和我的孩子说话了。",
+  "quark.misc.my_tomato": "不要再跟我和我的土豆说话了。",
   "quark.misc.my_children": "不要再跟我和我的孩子们说话了。",
   "quark.misc.shared_item": "%s分享了一件物品：",
   "quark.misc.configure_quark_here": "在这里配置夸克！",
+  "quark.misc.only_one_quill": "已经有一支寻路羽毛笔在搜索了……",
+  "quark.misc.quill_finished": "一幅寻路地图完成了。",
+  "quark.misc.quill_failed": "有一支寻路羽毛笔找不到附近有对应的生物群系。",
+  "quark.misc.quill_retry": "有一支寻路羽毛笔找不到附近有对应的生物群系。到别的地方再试试吧。",
```

如何翻译更通顺？

---

```suggestion
  "quark.misc.quill_failed": "寻路羽毛笔无法在附近找到对应的生物群系。",
  "quark.misc.quill_retry": "寻路羽毛笔无法在附近找到对应的生物群系。到别的地方再试试吧。",
```

一次只能有一支羽毛笔进行寻路，所以“有一支”可以删掉

---

我记得原文也是没有数量，后来加了数量，当时我之所以跟着加了一支这种数量词，隐隐约约记得是跟什么配置有关，有空再看看